src/collection/collection.const.ts

```diff
@@ -64,4 +64,8 @@ export const GET_MARCOM_ENTRY_TABLE_COLUMNS: () => TableColumn[] = () => [
 		label: i18n.t('collection/collection___link'),
 		id: 'external_link',
 	},
+	{
+		label: i18n.t('Acties'),
```

Comment:
the alignment of the 'Actions' column header and the button, doesn't match (left aligned header, vs right aligned button)

In the admin panel we just use an empty string for the actions column:
```
label: '',
```

You could the same here, unless the Actions label was a requirement in the jira task?